nari/ext/act/parser/ability.py

```diff
@@ -1,5 +1,6 @@
 """Parse ability (action) data from ACT log line"""
 from struct import unpack
+from typing import Tuple, List
```

Comment:
Don’t do this - python 3.10 has `tuple` and `list` as valid annotations now instead of importing from typing.

Reply:
Old imports that i didn't remove